test/helpers/cons.go

```diff
@@ -318,7 +318,7 @@ var ciliumCLICommands = map[string]string{
 	"cilium status --all-controllers":       "status.txt",
 	"cilium kvstore get cilium --recursive": "kvstore_get.txt",
 
-	"hubble observe --since 4h -o json": "hubble_observe.txt",
+	"hubble observe --since 4h -o jsonpb": "hubble_observe.json",
```

Comment:
Is `.json` the correct file extension for this format or is there some dedicated file extension?

Reply:
`.json` is correct extension. We just named the output type `jsonpb` because we didn't want to break the existing `json` output format.